src/routes/organizations/organizations.controller.ts

```diff
@@ -69,6 +70,26 @@ export class OrganizationsController {
     });
   }
 
+  @Post('/create-with-user')
+  @ApiOkResponse({
+    description: 'Organizations created',
+    type: CreateOrganizationResponse,
+  })
+  @ApiForbiddenResponse({ description: 'Forbidden resource' })
+  @ApiUnauthorizedResponse({ description: 'Signer address not provided' })
+  public async createWithUser(
+    @Body(new ValidationPipe(CreateOrganizationSchema))
+    body: CreateOrganizationDto,
+    @Auth() authPayload: AuthPayload,
+  ): Promise<CreateOrganizationResponse> {
+    return await this.organizationsService.createWithUser({
+      authPayload,
+      name: body.name,
+      status: getEnumKey(OrganizationStatus, OrganizationStatus.ACTIVE),
+      userStatuus: getEnumKey(UserStatus, UserStatus.ACTIVE),
```

Comment:
There's a spelling mistake (I'll only mark it here but will need to be propagated).

```suggestion
      status: getEnumKey(OrganizationStatus, OrganizationStatus.ACTIVE),
      userStatus: getEnumKey(UserStatus, UserStatus.ACTIVE),
```

Reply:
Nit: the string values are sufficient here, and are type safe.

Reply:
Fixed the typo. Let's fetch it from the enum.
https://github.com/safe-global/safe-client-gateway/pull/2344/commits/8ffc8a53505c566f8d26e322f155516353c23c9a